huntsman/camera/pyro.py

```diff
@@ -207,23 +211,23 @@ def take_exposure(self,
         # Make sure proxy is in async mode
         Pyro4.asyncproxy(self._proxy, asynchronous=True)
 
-        #Start the exposure
+        # Start the exposure
         self.logger.debug(f'Taking {seconds} second exposure on {self.name}: {filename}')
 
-        #Remote method call to start the exposure
+        # Remote method call to start the exposure
         exposure_result = self._proxy.take_exposure(seconds=seconds,
                                                     filename=filename,
                                                     dark=bool(dark),
                                                     *args,
                                                     **kwargs)
 
-        #Start a thread that will set an event once exposure has completed
-        exposure_thread = Timer(interval=get_quantity_value(seconds, u.s) + self.readout_time,
-                                function=self._async_wait,
-                                args=(exposure_result,
-                                      'exposure',
-                                      self._exposure_event,
-                                      self._timeout))
+        # Start a thread that will set an event once exposure has completed
+        exposure_thread = Thread(target=self._async_wait,
+                                 args=(exposure_result,
+                                       'exposure',
+                                       self._exposure_event,
+                                       get_quantity_value(seconds, u.s) +
+                                       self.readout_time + self._timeout))
```

Comment:
Maybe break this out into a variable before hand just so it's explicit: `total_exptime = ... + ...`

Also make me wonder if we shouldn't just have a property or method that returns that value (different PR).